projects/plugins/boost/app/assets/src/js/pages/settings/sections/Modules.svelte

```diff
@@ -188,17 +189,46 @@
 		</Module>
 	</div>
 
-	<Module slug="minify">
-		<h3 slot="title">{__( 'Minify', 'jetpack-boost' )}<span class="beta">Beta</span></h3>
+	<Module slug="minify_js">
+		<h3 slot="title">{__( 'Concatenate JS', 'jetpack-boost' )}<span class="beta">Beta</span></h3>
 		<p slot="description">
-			<TemplatedString
-				template={__(
-					`Minimize code and markup in your web pages and script files, reducing file sizes and speeding up your site. Read more on <link>web.dev</link>.`,
-					'jetpack-boost'
-				)}
-				vars={externalLinkTemplateVar( minifyCssLink )}
+			{__(
+				'Scripts are grouped by the original placement to reduce site loading time and lower number of requests.',
```

Comment:
Language suggestion.

```suggestion
				'Scripts are grouped by their original placement, concatenated and minified to reduce site loading time and reduce the number of requests.',
```